README.md

```diff
@@ -41,6 +43,12 @@ Please see below for the documentation relevant to you.
 - ❗️ All existing PRs against master will need to be closed and re-opened against 0-10-stable, if so desired
 - ❗️ Master, for the moment, won't have any released version of AMS on it.
 
+### Alternatives
+
+- [jsonapi-rb](http://jsonapi-rb.org/) is a [highly performant](https://gist.github.com/NullVoxPopuli/748e89ddc1732b42fdf42435d773734a) and modular JSON:API-only implementation.  There's a vibrant community around it that has produced projects such as [JSON:API Suite](https://jsonapi-suite.github.io/jsonapi_suite/).
+
+For benchmarks against alternatives, see https://github.com/rails-api/active_model_serializers/tree/benchmarks
+
 *Changes to 0.10.x maintenance*:
 
 - The 0.10.x version has become a huge maintenance version.  We had hoped to get it in shape for a 1.0 release, but it is clear that isn't going to happen.  Almost none of the maintainers from 0.8, 0.9, or earlier 0.10 are still working on AMS. We'll continue to maintain 0.10.x on the 0-10-stable branch, but maintainers won't otherwise be actively developing on it
```

Comment:
s/version/burden

lolz

Reply:
:-D

Reply:
I should probably actually have a different subheader for this stuff